backend/WebApplication1/Infrastructure/Utils/SearchReasonMap.cs

```diff
@@ -11,7 +11,7 @@ public static class SearchReasonMap
         public static Dictionary<SearchReason, string> reasonDictionary = 
```

Comment:
Good idea to use dict here.

Reply:
Im actually not sure about it, its pretty bulky, but I can't think of anything better, since Enum.GetName can't be used (NotMissing should be separated)

Reply:
The only way other way i thought of, was extension methods.
Maybe that?

Reply:
Good idea, let's write that down in trello and look at it later on, cuz we have other priorities now.